### DISCUSSION TITLE
UMEP URock tuto issue : java exception

### DISCUSSION THREAD

**Discussion by pdugue** [UMEP-dev/UMEP#534] [Q&A]
Hello !
I have started using URock by following the tuto available here : https://umep-docs.readthedocs.io/projects/tutorial/en/latest/Tutorials/IntroductionToURock.html 
I have followed these steps without issues : URock Prepare and URock.
But with the next step, URock Analyzer, this error appeared : 
![error](https://github.com/UMEP-dev/UMEP/assets/138692633/b476e93d-1b6e-48ad-9727-1d4f54d392fa)

Do you know why this error occurs and how to solve it ?
Thanks, best, 
Pauline

**Comment by j3r3m1** [comment]
Hi @pdugue the problem comes from the fact that only shape and geojson files can be used as input (you have used geopackage). Using shapefile or geojson will solve the problem.

**Reply by pdugue** [reply]
Hi @j3r3m1, thanks for your answer, it has been really helpful, I don't have the error anymore !

**Comment by j3r3m1** [comment]
cool perfect !

**Reply by pdugue** [reply]
Hi, 

As mentionned before, I don't have the error anymore. However, nothing appears on QGIS (no layer, no data).
How can we find the results of the URock Analyzer ?
![image](https://github.com/UMEP-dev/UMEP/assets/138692633/f18768f4-4444-4638-a196-b029cdb4b2d7)

Thanks, best,

Pauline

**Comment by j3r3m1** [comment]
Figures are saved in the 'OUTPUT_DIRECTORY' folder which is given in the image you have submitted. You should have one figure by line provided.
